### PR TITLE
Fix observatory card flip. Bootstrap integration tests with chromedriver.

### LINKED ISSUE #2189: Can't turn observatory cards
https://gobiernoabierto.getafe.es/observatorio

### PR DESCRIPTION
**Pull request by amiedes**
Closes #2189 

## :v: What does this PR do?

- Fixes a markup bug that made the observatory cards not turning
- Sets up a helper for progessively start using chromedriver
- Sets up a test helper for mocking Populate Data requests

## :book: Does this PR require updating the documentation?

You need `chromedriver` in local:

```bash
brew cask install chromedriver
```

Development docs updated [here](https://gobierto.readme.io/v0.1/docs/development-environment-1#section-integration-tests-drivers).

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -84,6 +84,17 @@ class Application < Rails::Application
     if ENV["GOBIERTO_ROOT_URL_PATH"].present?
       Rails.application.config.relative_url_root = ENV["GOBIERTO_ROOT_URL_PATH"]
     end
+
+    if Rails.env.test?
```
Comment (ferblape) [inline config/application.rb]: Could you move this to config/environments/test.rb ??

**Comment by amiedes** [comment]
Changes addressed, deployed to [staging](http://esplugues.gobify.net/observatorio) and updated docs and PR description

**Comment by ferblape** [comment]
Could you review the failing test?

**Comment by ferblape** [comment]
You need to update circleci config to add chromedriver: `Selenium::WebDriver::Error::WebDriverError:  Unable to find chromedriver. Please download the server from http://chromedriver.storage.googleapis.com/index.html and place it somewhere on your PATH. More info at https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver.`

**Comment by amiedes** [comment]
Fixed

**Pull request merged**